cmd/createtree/pkcs11.go

```diff
@@ -1,64 +0,0 @@
-// +build pkcs11
```

Comment:
Are there any more files left with this build tag?

There is a CI testing target using it. Should it be removed too?

Reply:
There's very little: there's the packages in `crypto/keys/pkcs11`, which are still imported by the mains.

I intend on cleaning that up separately, especially as this will impact the CI (and that can have its own complications!)...

Reply:
Oh, and there's a possibility that this is still used by CTFE!